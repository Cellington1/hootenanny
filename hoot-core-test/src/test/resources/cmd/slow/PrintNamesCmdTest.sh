#!/bin/bash
set -e

#mkdir -p $HOOT_HOME/test-output/cmd/slow/

inputfile=$HOOT_HOME/test-files/DcGisRoads.osm

hoot names $inputfile
