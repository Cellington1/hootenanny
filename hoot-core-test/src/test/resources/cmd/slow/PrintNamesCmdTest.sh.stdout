Total word count: 1163
320 (0.28) : nw
246 (0.21) : st
80 (0.069) : ave
45 (0.039) : sw
18 (0.015) : h
18 (0.015) : pennsylvania
17 (0.015) : i
17 (0.015) : 14th
16 (0.014) : e
15 (0.013) : constitution
15 (0.013) : 17th
14 (0.012) : dr
14 (0.012) : 15th
13 (0.011) : k
13 (0.011) : virginia
12 (0.01) : 18th
12 (0.01) : new
12 (0.01) : cir
11 (0.0095) : 25th
11 (0.0095) : 23rd
11 (0.0095) : independence
11 (0.0095) : f
11 (0.0095) : g
10 (0.0086) : 21st
10 (0.0086) : 20th
10 (0.0086) : 19th
9 (0.0077) : bn
9 (0.0077) : 13th
9 (0.0077) : c
9 (0.0077) : interstate
9 (0.0077) : 66
7 (0.006) : memorial
7 (0.006) : pl
7 (0.006) : 22nd
7 (0.006) : 12th
6 (0.0052) : york
6 (0.0052) : hampshire
6 (0.0052) : lincoln
6 (0.0052) : 24th
6 (0.0052) : washington
5 (0.0043) : d
5 (0.0043) : ohio
4 (0.0034) : executive
4 (0.0034) : madison
4 (0.0034) : 16th
3 (0.0026) : maine
3 (0.0026) : jefferson
3 (0.0026) : expy
3 (0.0026) : 26th
2 (0.0017) : connecticut
2 (0.0017) : rock
2 (0.0017) : rd
2 (0.0017) : raoul
2 (0.0017) : potomac
2 (0.0017) : pkwy
2 (0.0017) : state
2 (0.0017) : vermont
2 (0.0017) : creek
2 (0.0017) : east
2 (0.0017) : wallenberg
2 (0.0017) : ellipse
2 (0.0017) : &
2 (0.0017) : west
1 (0.00086) : arlington
1 (0.00086) : alexander
1 (0.00086) : annes
1 (0.00086) : south
1 (0.00086) : 13
1 (0.00086) : 12
1 (0.00086) : queen
1 (0.00086) : henry
1 (0.00086) : bacon
1 (0.00086) : basin
1 (0.00086) : brg
1 (0.00086) : daniel
1 (0.00086) : ln
1 (0.00086) : french
1 (0.00086) : jackson
1 (0.00086) : hamilton
